.gitignore

```diff
@@ -3,3 +3,4 @@
 *.swp
 *.swo
 third_party/*
+core/platform/protobuf/protofiles/*.py
```

Comment:
This seems a very general exception for .gitignore... It feels like it's not right?

Reply:
Actually, the `protofiles/*.py` are the `protoc` generated files which shouldn't be part of the repo because they are compiled on the fly for consistency with latest definitions. Hence, we need to include that exception. `protofiles` directory would contain only proto definitions and their compiled python files.

Reply:
Perhaps add a comment above this to explain?